src/wordlift/webhooks/api/class-rest-controller.php

```diff
@@ -28,32 +25,28 @@ public function __construct() {
      * @return json
      */
 
-	public function register_sync_many( $payloads, $object ) {
+	public function register_sync_many( $payloads ) {
 
         $webhook_urls = get_option('wl_webhook_url');
-        if( ! empty( $webhook_urls ) ) {
-            foreach( $webhook_urls as $webhook_url ) {
-                $handle = curl_init( $webhook_url ); //'https://ensh04p5m7cs4hw.m.pipedream.net';
-
-                $data = [
-                    'payload'   => $payloads,
-                    'object'    => $object
-                ];
-
-                $encodedData = json_encode( $data );
-
-                curl_setopt( $handle, CURLOPT_CUSTOMREQUEST, "POST" );
-               // curl_setopt($handle, CURLOPT_POST, 1);
-                curl_setopt( $handle, CURLOPT_POSTFIELDS, $encodedData );
-                curl_setopt( $handle, CURLOPT_RETURNTRANSFER, true );
-                curl_setopt( $handle, CURLOPT_HTTPHEADER, array(
-                    'Content-Type: application/json',
-                    'Content-Length: ' . strlen( $encodedData ) )
-                );
-
-                $result = curl_exec( $handle );
-                return $result;
-            }
+         if( empty( $webhook_urls ) ) {
+            return;
+         }
+
+        foreach( $webhook_urls as $webhook_url ) {
+
+            $data = array(
+                'payload'   => $payloads
+            );
+            $encoded_data = json_encode( $data );
```

Comment:
Because the payload is already a string, you you can just set it as body, no need to nest it and encode it